assignment/client/src/api/questionClient.js

```diff
@@ -3,14 +3,18 @@ import axios from "axios";
 const questionsApi =
   process.env.QUESTIONS_API_URL || "http://localhost:8888/questions";
 
+const getHeader = () => {return {headers: {
+  'Authorization': `Bearer ${localStorage.getItem('token')}`
+}}};
+
 export const getQuestions = async () => {
-  let { data } = await axios.get(questionsApi);
+  let { data } = await axios.get(questionsApi, getHeader());
   return data;
 };
 
 export const getQuestion = async (id) => {
   try {
```

Comment:
gotta transfer the try catch here into react component to display authorization errors